inventory/rh-sso/hosts

```diff
@@ -9,3 +9,6 @@ rh-sso-hosts
 
 [rhsm_hosts:children]
 rh-sso-hosts
+
+[identity-hosts:children]
+rh-sso-hosts
```

Comment:
I've been thinking about this one a bit more as well. Could we add a note to the README that you can switch this to `localhost` if you don't have access to the SSO server, but would need to move the credentials `all.yml`?

Reply:
Absolutely. I can insert this piece of information there.